src/python/pants/backend/java/compile/javac_test.py

```diff
@@ -52,12 +54,14 @@ def rule_runner() -> RuleRunner:
             *util_rules(),
             *javac_binary_rules(),
             *target_types_rules(),
+            *fs_rules(),
```

Comment:
This shouldn't be necessary. Registered by default

Reply:
Removed.